docs/github.md

```diff
@@ -0,0 +1,41 @@
+# Using Jujutsu with Github Projects
+
+## Setup a ssh key
+As of December 2022 it's necessary to setup a ssh key to work with github
+projects. See [Github's Tutorial][gh]. This restriction may be lifted in the 
+future, see issue #469 for more information and progress on authenticated http. 
+
+
+## Basic workflows
+Jujutsu supports both a `git` and a `hg` like workflows. This allows you
+to continue using your favorite way of using the vcs. It's also possible to 
+just commit multiple changes and set the expected branch retroactively. 
+
+### Git workflow
+
+```shell script
+$ jj branch c $your-feature
```

Comment:
nit: Spell this out as `jj branch create`? The "c" probably seems quite cryptic to new users

Reply:
Done.